index.html

```diff
@@ -4,9 +4,68 @@
+                <a href="https://www.linkedin.com/login"> <img class="logos" src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/81/LinkedIn_icon.svg/2048px-LinkedIn_icon.svg.png" alt="LinkedIn logo"></a>
+            </nav>
+        </div>
+
+    </header>
     
+
+
+    <section class="Your-tasks">
+
+        <form action="#" method="post">
+            <fieldset>
+                <h2>Your Tasks</h2>
+                <input type="text" name="task-name" id="task-name" placeholder="e.g. wash dishes">
+                <button class="violet-button" type="submit">Add</button>
+                <p>There Was an error</p>
+            </fieldset>
+        </form>
+
+    </section>
+
+    <section class="lists">
+
+        <div class="trashes" >
+            <h3>Take Out Trashes</h3>
```

Comment:
I wouldn't call these headings. The task names don't have any semantic value as h3s.